subworkflows/local/bgc.nf

```diff
@@ -131,6 +151,13 @@ workflow BGC {
         BGC_HMMER_HMMSEARCH ( ch_in_for_bgc_hmmsearch )
         ch_versions = ch_versions.mix(BGC_HMMER_HMMSEARCH.out.versions)
     }
+
+    // COMBGC
+    COMBGC ( ch_bgcresults_for_combgc )
+
+    ch_combgc_summaries = COMBGC.out.tsv
+        .collectFile(name: 'combgc_complete_summary.csv', storeDir: "${params.outdir}/reports/combgc", keepHeader:true)
```

Comment:
To have it right in the output.md: will you have a summary for each sample plus the complete_summary in the output folder?

Reply:
Exactly. Each sample gets an output in:
`reports/combgc/sampleX/combgc_summary.tsv`, and there is the concatenated version in:
`reports/combgc/combgc_complete_summary.tsv`